javascript/javascript1/week4/week4.js

```diff
@@ -0,0 +1,103 @@
+let name;
+const todo = [];
+function getReply(command)
+{
+  const arrayOfStrings = command.split(' ');
+  const num1 =  Number(arrayOfStrings[2]);
+  const num2 =  Number(arrayOfStrings[4]); 
+  switch(command){
+    case 'hello my name is Benjamin':
+      name = command.substring(17);
```

Comment:
Try if you can split the string and then use array index to access the name in the command.